package.json

```diff
@@ -41,6 +41,7 @@
     "@types/react-router-dom": "^5.1.5",
     "@types/styled-components": "^5.1.7",
     "bignumber.js": "^9.0.0",
+    "canvas-confetti": "^1.3.3",
```

Comment:
Given the api on this there is no point in my opinion to put this in the UI Kit.

Reply:
agree